app/views/directives/deploy-image.html

```diff
@@ -203,7 +203,7 @@ <h2>
         Could not load image metadata.
       </h2>
       <p>{{import.error | upperFirst}}</p>
-      <p>The image may not exist or it may be in a secure registry. Check that you have entered the image name correctly or <a href="" ng-click="openCreateWebhookSecretModal()">create an image pull secret</a> with your image registry credentials and try again.</p>
+      <p>The image may not exist or it may be in a secure registry. Check that you have entered the image name correctly. In case of private image repository, check that your image pull secret credentials are correct or <a href="" ng-click="openCreateWebhookSecretModal()">create an image pull secret</a> with your image registry credentials and try again.</p>
```

Comment:
Since the API server error already says the username/password is incorrect, I don't see much value in restating that. And if we always state it, we'll say it even when that's not the case, whereas the error message from the server will only appear when it's really a problem.

I'd rather close the bug as won't fix.

Reply:
Seems good to me. Will close PR and bug as well.
Thanks !